e2e-tests/jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    preset: "@hover/jest-playwright-preset",
```

Comment:
I spent some time deciding and ended up going with `jest-playwright` for the following reasons:

- I wanted to standardize the queries, assertions and hooks between our e2e code and our unit and integration test. That way we can move across them without having to learn / review a new API
- `@playwright/test` also does not currently have support for `testing-library` queries that will allow us to closely simulate user behaviour in our e2e tests. This is what broke the tie for me. Also the `testing-library` queries are the same selectors we will be using in our unit and integration tests.

Reply:
1. You need to use `yarn add` to add these so that the `yarn.lock` file is updated
2. These should actually be primary `dependencies`, not `devDependencies`, because these packages are actually part of the primary use case for this package -- to test the wallet end-to-end :). In a standard package where these dependencies only exist to be used when running internal tests against the code in `e2e-tests`, then they'd be dev dependencies...but that's not the case here :)

Reply:
We hashed this out via voice calls :+1: